### ISSUE TITLE
unseen activities IconBadge doesn't disappear after marking activities as seen in DropdownNotification

### ISSUE THREAD

**Issue by j86hughes** [GetStream/react-activity-feed#102]
I am using the NotificationDropdown component. When I have unseen activities in the feed, the IconBadge is displayed over the bell icon with the number of unseen activities. I expect the IconBadge to disappear when clicking on the bell icon, as I am calling onMarkAsSeen for each Group rendered in the NotificationFeed, however the IconBadge doesn't disappear unless I manually refresh the page.

**Comment by vivaldi-va** [comment]
I've looked into this a bit and believe I've found the reason for this, and other similar issues. From what I can tell, the issue stems from having multiple `FeedManager` contexts nested, and becoming out of sync from one another. Thus, updating with `mark_seen` for example, will cause the state on the child context, but not the parent.

e.g. using a rough copy of the `NotificationDropdown` component:

```jsx
<NotificationDropdown>
  <FeedCtx>
    <IconBadge />
  </FeedCtx>

  <NotificationFeed>
    <FeedCtx>
      { /* ... */ }
    </FeedCtx>
  </NotificationFeed>
</NotificationDropdown>
```

The context surrounding the icon badge will not be updated if notifications within the NotificationFeed are updated with `mark_seen`, and thus the icon badge count will not update. 

Not sure what the best solution for it could be, however. One solution I had in mind was to re-use an existing context if it exists, rather than creating a new `FeedManager` each time. This should then keep only a single copy of the state to be shared by all it's child components.

Happy to hear any thoughts on this
